src/main/java/uk/gov/pay/directdebit/common/exception/InternalServerErrorExceptionMapper.java

```diff
@@ -16,7 +16,15 @@
 
     @Override
     public Response toResponse(InternalServerErrorException exception) {
-        LOGGER.error(exception.getMessage());
+        StringBuilder sb = new StringBuilder();
```

Comment:
Would it be simpler to pass the throwable to the logger along with the message we want to print and let it print the stack trace which prints the causes along with the classes and line numbers which might be helpful `LOGGER.error(exception.getMessage(), exception.getCause());` or is that too noisy?

Reply:
If we log the stack trace it gets split across multiple lines, which is painful to deal with in Splunk. When we have structured logging, we’ll be able to include stacktraces much more easily and we should revisit this then.

Reply:
OK, agree it being split makes it harder but was thinking that avoiding this extra code and having the class/line numbers made it worth it. If you want to go this route whilst we wait then ok 👍